scripts/kfctl.sh

```diff
@@ -15,9 +15,14 @@ WHAT=$2
 
 ENV_FILE="env.sh"
 
+KUBEFLOW_VERSION=${KUBEFLOW_VERSION:-"master"}
+KUBEFLOW_DEPLOY=${KUBEFLOW_DEPLOY:-true}
+K8S_NAMESPACE=${K8S_NAMESPACE:-"kubeflow"}
```

Comment:
Why do we need K8S_NAMESPACE to be provided as an environment variable?
This should probably be a parameter of kfctl init if its really needed

Reply:
This is already defined in CREATE_ENV

Reply:
removed K8S_NAMESPACE here as well